idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -503,7 +492,7 @@ def initialize(self, state_args={}, outlvl=idaeslog.NOTSET, solver=None,
 
         with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
             res = opt.solve(self, tee=slc.tee)
-        self.control_volume.release_state(flags=flags, outlvl=outlvl)
+        self.control_volume.release_state(flags={}, outlvl=outlvl)
```

Comment:
You don't need to call `release_state` here, as you didn't set `hold_state=True` when you called the `initialize` method above.

Reply:
Hi @andrewlee94 : Could you review the model again? Thanks.